test/is-sql-tests/node-sql-parser.test.ts

```diff
@@ -0,0 +1,276 @@
+import { Parser } from 'node-sql-parser';
+
+// The isolated is-sql implementation:
```

Comment:
We should extract the is-sql implementation into its own function, so that we can call it from `assertions.ts` and from its test.  This means we don't have to maintain two separate versions of the logic, which will invariably fall out of sync.

Reply:
got it